src/data/consul-landing.json

```diff
@@ -15,7 +15,7 @@
       "text": "In this guide we cover what Consul is, what problems it can solve, how it compares to existing software, and how you can get started using it.",
       "link": {
         "text": "Learn More",
-        "url": "https://www.consul.io/docs/intro"
+        "url": "/consul/docs/intro"
```

Comment:
Have updated all URLs in `consul-landing`, these routes are now accessible within `dev-portal` 🎉